app/Http/Requests/Api/SortApiPostRequest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Requests\Api;
+
+use Illuminate\Foundation\Http\FormRequest;
+use App\Post;
+
+class SortApiPostRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'sort' => 'in:updated_at',
+            'sort_type' => 'in:DESC,ASC',
+            'type' => 'in:' .implode(',', [Post::COMMENT, Post::REVIEW]),
```

Comment:
vì sao phải làm như vậy em:
`'type' => 'in:' . Post::COMMENT . ',' . Post::REVIEW,`